docs/management/index-patterns.asciidoc

```diff
@@ -20,96 +25,103 @@ image::images/management-index-read-only-badge.png[Example of Index Pattern Mana
+that you set up for {ccs}, which are named `cluster_one` and `cluster_two`,
+you would use `cluster_one:logstash-*,cluster_two:logstash-*` as your index pattern.
 
-See <<visualize-rollup-data, Creating a visualization using rolled up data>> 
-for more detailed information.
+You can use wildcards in your cluster names
+to match any number of clusters, so if you want to search {ls} indices across
+clusters named `cluster_foo`, `cluster_bar`, and so on, you would use `cluster_*:logstash-*`
+as your index pattern.
+
+To query across all {es} clusters that have been configured for {ccs},
+use a standalone wildcard for your cluster name in your index
+pattern: `*:logstash-*`.
+
+Once an index pattern is configured using the {ccs} syntax, all searches and
+aggregations using that index pattern in {kib} take advantage of {ccs}.
 
 [float]
 === Manage your index pattern
 
-Once you’ve created an index pattern, you’re presented a table of all fields 
-and associated data types in the index. 
+Once you create an index pattern, manually or with a sample data set,
+you might want to look at its fields and associated data types.
+You might also want to perform housekeeping tasks, such as making the
+index pattern the default or deleting it when you longer need it.
+You can drill down into the details of an index pattern by clicking its name in
+the *Index patterns* overview.
 
 [role="screenshot"]
 image:management/index-patterns/images/new-index-pattern.png["Index files and data types"]
 
-You can perform the following actions:
+From this view, you can perform the following actions:
 
-* *Manage the index fields.* Click a column header to sort the table by that column. 
-Use the field dropdown menu to limit to display to a specific field. 
-See <<managing-fields, Managing fields>> for more detailed information. 
+* *Manage the index fields.* Click a column header to sort the table by that column.
+Use the field dropdown menu to limit the display to a specific field.
+See <<managing-fields, Managing fields>> for more detailed information.
 
-* [[set-default-pattern]]*Set the default index pattern.* {kib} uses a badge to make users 
-aware of which index pattern is the default. The first pattern 
-you create is automatically designated as the default pattern. The default 
-index pattern is loaded when you view the Discover tab.  
+* [[set-default-pattern]]*Set the default index pattern.* {kib} uses a badge to make users
+aware of which index pattern is the default. The first pattern
+you create is automatically designated as the default pattern. The default
+index pattern is loaded when you open *Discover*.
 
-* [[reload-fields]]*Reload the index fields list.* You can reload the index fields list to 
-pick up any newly-added fields. Doing so also resets Kibana’s popularity counters 
-for the fields. The popularity counters keep track of the fields 
+* [[reload-fields]]*Reload the index fields list.* You can reload the index fields list to
```

Comment:
In the UI I believe we say "refresh" when you hover over the icon, instead of "reload." 

Not sure if using the same terminology in the docs matters, but thought I'd point that out.